dominion/dominion_ui/src/components/ReleaseDropdown.js

```diff
@@ -123,17 +124,17 @@ class ReleaseDropdown extends Component {
           open={menuOpen}
           onClose={this.closeReleaseMenu}
         >
-          <Paper>
-            <ClickAwayListener onClickAway={this.closeReleaseMenu}>
+          <ClickAwayListener onClickAway={this.closeReleaseMenu}>
+            <Paper>
```

Comment:
Did this order solve the click away issue ? @kilemensi

Reply:
Yep @karimkawambwa

Reply:
Tested it 👍